src/js_command_match/keys/list.rs

```diff
@@ -0,0 +1,71 @@
+use crate::js_command_match::constants::NETWORK_ID_ALIASES;
+
+#[derive(Debug, Clone, clap::Parser)]
+#[clap(alias("list-keys"))]
```

Comment:
Since the command in the top-level enum included as `ListKeys`, it is already expecting `list-keys` command, so this alias only causes a parsing error:

```
env RUST_BACKTRACE=1 cargo run -- validators
```

<img width="1082" alt="image" src="https://github.com/near/near-cli-rs/assets/304265/a5d86e62-cfd9-430d-b09c-886c052c22ad">

Consider removing the alias or if there is a different alias, use it here.

```suggestion
```